scipy/stats/mstats_basic.py

```diff
@@ -2152,10 +2152,26 @@ def variation(a, axis=0):
     -------
     variation : ndarray
         The calculated variation along the requested axis.
-
+    
     Notes
     -----
     For more details about `variation`, see `stats.variation`.
+    
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
I think you can leave this line out as it is performed by our refguide machinery.